### DISCUSSION TITLE
Need confirmation, enumate() here looks useless as index is ignored right after

### DISCUSSION THREAD

**Discussion by adjabaev** [DioxusLabs/taffy#299] [Ideas]
I don't know if this is the right place, I felt like an issue was a little bit too much

https://github.com/DioxusLabs/taffy/blob/e2cee9d6f3e4f8ebff7d0482b90c6c2b8201d3fd/src/compute/flexbox.rs#L1558

**Comment by nicoburns** [comment]
The index finds it's way through as `order` here https://github.com/DioxusLabs/taffy/blob/e2cee9d6f3e4f8ebff7d0482b90c6c2b8201d3fd/src/compute/flexbox.rs#L1562

**Reply by adjabaev** [reply]
Tuples are incredible!
Thank you for the explanation 😊
